solutions/instances/variables.tf

```diff
@@ -116,7 +249,13 @@ variable "enable_at_event_routing_to_cos_bucket" {
 
 variable "enable_at_event_routing_to_log_analysis" {
   type        = bool
-  description = "Whether to enable event from Activity Tracker to Log Analysis."
+  description = "Whether to enable event routing from Activity Tracker to Log Analysis."
+  default     = false
+}
+
+variable "enable_at_event_routing_to_cloud_logs" {
+  type        = bool
+  description = "Whether to enable event routing from Activity Tracker to Cloud Log."
   default     = false
```

Comment:
We probably wan't to default this to `true` now and set `enable_at_event_routing_to_cos_bucket` to `false`, but perhaps run this by the team first

Reply:
Further discussion about this - https://ibm-cloudplatform.slack.com/archives/CG06FQ31P/p1726660942244999F